src/runtime/experiments.ts

```diff
@@ -194,20 +182,17 @@ function parseSetExperiment(win, experimentMap, spec) {
     } catch (e) {
       // Ignore: experiment parsing cannot block runtime.
       on = false;
-      ErrorUtils.throwAsync(e);
+      ErrorUtils.throwAsync(e as Error);
     }
   } else {
     on = false;
   }
 
+  experimentMap ||= {};
```

Comment:
Can you change the type of experimentMap so it is not nullable, instead?

Reply:
Updated ✅